src/it/java/net/spy/memcached/MemcachedClientConstructorTest.java

```diff
@@ -37,6 +37,8 @@
 import java.util.List;
 import java.util.Map;
 
+import javax.net.ssl.SSLContext;
```

Comment:
Do you need to change this class at all?

Reply:
The Dynamic mode test will kick off all the tests in MemcachedClientConstructorTest class. We need to have this change in TLS mode.